README.md

```diff
@@ -1,6 +1,6 @@
 # boot-sassc
 
-[![Clojars Project](http://clojars.org/mathias/boot-sassc/latest-version.svg)](http://clojars.org/mathias/boot-sassc)
+[![Clojars Project](http://clojars.org/boot-sassc/latest-version.svg)](http://clojars.org/mathias/boot-sassc)
```

Comment:
Yes, it should !

Just look at them side by side:

this is the previous one
http://clojars.org/mathias/boot-sassc

This is the new one
http://clojars.org/mathias/boot-sassc

Do you see the difference ?